### DISCUSSION TITLE
Orchard Core CMS  Add Custom Login Form

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#14915] [Q&A]
Hi  I am new in orchard core CMS and I need to  add a custom login form on my project theme can someone give me the process to add this custom form step by step. 
thank you

**Comment by hishamco** [comment]
You can override the login page located at https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml

You remind me of this issue https://github.com/OrchardCMS/OrchardCore/issues/3473 which closed at that time, feel free to see what I did in Orchard Core Contrib (OCC) [here](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/tree/main/src/OrchardCoreContrib.Themes.Admin/Views/OrchardCore.Users/Account)

**Reply by YoumnaJ** [reply]
thank you  for you help but I am creating the code from scratch and I don't have all orchard Core module If someone can provide me with a solution for my problem and I am using agency theme only

**Reply by hishamco** [reply]
Don't worry about everything else, you just need to override the view in your own theme

**Reply by YoumnaJ** [reply]
Ok thank you it's necessary to put the new custom login under modules in the folder of orchardcore.User? And we should add something to startup project to modify the path of the default login?

**Reply by sambeauvois** [reply]
Hi,

I was looking for a way to orverride the login page or layout from a "front" theme.

I had my "aha" moment with this page of doc :

https://docs.orchardcore.net/fr/latest/docs/reference/modules/Templates/

> 
> Overriding Login view[¶](https://docs.orchardcore.net/fr/latest/docs/reference/modules/Templates/#overriding-login-view)
> For example, if you want to override the OrchardCore.Users\Views\Account\Login.cshtml view you would need to create a file in your theme and place it under YourTheme\Views\OrchardCore.Users\Account\Login.cshtml.
> For this particular file, you would also need to select the Use site theme for login page option under the Configuration->Login page in the admin.

It's not in "configuration>login" but in in /admin>Security>Settings>User Login
